RecoTracker/MeasurementDet/plugins/MeasurementTrackerEventProducer.h

```diff
@@ -36,6 +39,8 @@ class dso_hidden MeasurementTrackerEventProducer final : public edm::stream::EDP
       edm::EDGetTokenT<edm::ContainerMask<edmNew::DetSetVector<SiStripCluster>>> theStripClusterMask;
 
       std::vector<edm::EDGetTokenT<DetIdCollection>>      theInactivePixelDetectorLabels;
+      std::vector<edm::EDGetTokenT<PixelFEDChannelCollection> > theBadPixelFEDChannelsLabels;
+      std::string pixelCablingMapLabel_;
```

Comment:
it would be nice to keep a consistent naming pattern for data members: ```thePixelCablingMapLabel```

Reply:
Kept consistency with line 33:
std::string measurementTrackerLabel_;